lib/DirectAdmin_Domain.php

```diff
@@ -44,7 +44,7 @@ public function __construct($domain, $username, $pointsto = null){
 	 */
 	public function isRemote(){
 		$api = new DirectAdmin_API();
-		$records = $api->getDomainsMxRecords($this->domain);
+		$records = $api->getDomainsMxRecords($this->domain, $this->username);
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->